README.md

```diff
@@ -449,7 +449,7 @@ When using `catch`, or passing a [rejection callback](https://developer.mozilla.
 
 ## Config Defaults
 
-You can specify config defaults that will be applied to every request.
+You can specify config defaults that will be applied to every request, although there are some fields which will override your defaults if passed at the time of the request.  See [Config order of precedence](#config-order-of-precedence)
```

Comment:
See [Config order of precedence](#config-order-of-precedence) for more details about config merging strategies.